### PR TITLE
Add server console output

### PR DESCRIPTION
**Pull request by casperboone**
This PR adds the ability to get the console output of a server (see [OpenStack docs](https://developer.openstack.org/api-ref/compute/#show-console-output-os-getconsoleoutput-action) ).

### CODE REVIEW

File: src/Compute/v2/Models/Server.php [hunk 1]
```diff
@@ -239,6 +239,22 @@ public function revertResize()
     }
 
     /**
+     * Gets the console output of the server.
+     *
+     * @param int $length The number of lines, by default all lines will be returned.
+     * @return string
+     */
+    public function getConsoleOutput(int $length = -1)
```
Comment (laszlof) [inline src/Compute/v2/Models/Server.php]: Could use a return typehint

Comment (casperboone) [reply src/Compute/v2/Models/Server.php]: Thanks, done.

File: src/Compute/v2/Models/Server.php [hunk 2]
```diff
@@ -239,6 +239,22 @@ public function revertResize()
     }
 
     /**
+     * Gets the console output of the server.
+     *
+     * @param int $length The number of lines, by default all lines will be returned.
+     * @return string
+     */
+    public function getConsoleOutput(int $length = -1): string
```
Comment (haphan) [inline src/Compute/v2/Models/Server.php]: What is the logic behind default value`length = -1`?

**Comment by haphan** [comment]
@casperboone Only have one small comment for you. Otherwise all looks good.

File: src/Compute/v2/Models/Server.php (hunk 2 above)
Comment (haphan) [reply src/Compute/v2/Models/Server.php]: > When given a length of -1, the API will return all lines

This is an assumption thus subject to change in the future; I think we better follow official docs that `length` shall not be included in the request if we want API to return all lines.

Comment (casperboone) [reply src/Compute/v2/Models/Server.php]: Good point, will make that change soon 😄.

**Comment by casperboone** [comment]
@haphan Updated the implementation, could you please have another look? 😄

**Comment by haphan** [comment]
@casperboone Can you rebase with master branch?

**Comment by casperboone** [comment]
@haphan Done! :)

**Pull request merged**